Kudu.Services/Filters/RequestThrottleActionFilterAttribute.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+
+namespace Kudu.Services.Filters
+{
+    public sealed class RequestThrottleActionFilterAttribute : ActionFilterAttribute
+    {
+        int _concurrency = 0;
+
+        public int MaxConcurrency { get; set; } = 5;
```

Comment:
Wonder if this should be appSettings with upper limit at, say, 20 pending requests.  I understand bad usages can abuse it (they can anyway by deploying private kudu).  On the flip side, it gives us a quick knob to mitigate if regression.     We can ship with appSettings and then look at log if normal usage will ever get to this and make it const.